junifer/markers/falff/tests/test_falff_spheres.py

```diff
@@ -0,0 +1,162 @@
+"""Provide test for parcel-aggregated (f)ALFF."""
+
+# Authors: Federico Raimondo <f.raimondo@fz-juelich.de>
+#          Synchon Mandal <s.mandal@fz-juelich.de>
+# License: AGPL
+
+import pytest
+
+from pathlib import Path
+
+from numpy.testing import assert_array_equal
+from scipy.stats import pearsonr
+
+from junifer.datareader import DefaultDataReader
+from junifer.markers.falff import AmplitudeLowFrequencyFluctuationSpheres
+from junifer.testing.datagrabbers import PartlyCloudyTestingDataGrabber
+from junifer.pipeline.utils import _check_afni
+from junifer.storage import SQLiteFeatureStorage
+from junifer.utils import logger
+
+
```

Comment:
As the coordinates used are same, maybe create a module-level variable and reuse it.